backend/typescript/services/implementations/camperService.ts

```diff
@@ -127,6 +127,77 @@ class CamperService implements ICamperService {
 
     return camperDtos;
   }
+
+  async deleteCampersByChargeId(chargeId: string): Promise<void> {
+    try {
+      const numericalChargeId = Number(chargeId);
```

Comment:
I just checked the Stripe docs and chargeId is actually a string D: can we change the type to a string in the camper model and remove this?